datasets/eumetsat/dataset.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataset:
+  name: eumetsat
+  friendly_name: ~
+  description: ~
+  dataset_sources: ~
+  terms_of_use: ~
```

Comment:
Should these be empty?

Reply:
@leahecole Yup, they're expected to be empty placeholders for now until we launch the doc set feature.

Reply:
ack